doc/GIT_USAGE.md

```diff
@@ -19,23 +19,23 @@
 ## Branching Model
 
 - The DMOD repo uses a branching model based on [Gitflow](https://nvie.com/posts/a-successful-git-branching-model/) and has two primary long-term branch: 
-  - **master**: the main branch pointing to the current, official version
-  - **dev**: an integration branch containing the latest completed development work intended for the next released version
-- Most interaction with DMOD repo is done via pull requests (PRs) to the `dev` branch
-  - Independent branches for features or bug fixes are created off `dev` to contain development work that is in progress
-  - These branches are reviewed and their changes integrated back into `dev` once complete via PRs
+  - **master**: the main development and integration branch containing the latest completed development work intended for the next released version
+  - **production**: the branch representing the latest code verified as production-ready and pointing to the most recently release, official version
```

Comment:
Can you emphasize that 99% of contributors will not be involved with the production branch? Most devs _will_ be connected to the OWP, so they'd be an OWP contributor, but not an OWP versioner or maintainer.

I think this interaction has been what's caused the most confusion.